src/main/java/io/interop/finsemble/example/fdc3/JavaFDC3Example.java

```diff
@@ -66,6 +67,7 @@ public JavaFDC3Example(final String[] args) throws Exception {
      */
     private void configureFinsemble(final String[] args) {
         finsemble = new Finsemble(Arrays.stream(args).sequential().collect(Collectors.toList()), this);
+        finsemble.setAppName("JavaFDC3Example");
```

Comment:
Good practice to set the app name

Reply:
I made a change in the Finsemble lib to set the appName to the passed in `appId` (this was _supposed_ to happen but a bug prevented it from happening in some cases). The default value is now the name of the main class, as well.

Setting the appName in this example is helpful to work around a bug in Finsemble.jar, but that bug has been fixed so this line isn't necessary. I've removed it.